docs/arch/adr-001-links-based-facets.md

```diff
@@ -0,0 +1,90 @@
+# Tagging content from multiple publishers to a facet
+
+Date: 2019-06-26
+
+## Context
+
+In late 2018, the [business finder](https://www.gov.uk/find-eu-exit-guidance-business) was added to GOV.UK.
+This finder was different to the others for two reasons:
+
+1. It filtered on content published by multiple publishing apps.
+2. It had customised logic between the facets. The results between the first two facets were "subtractive", like a normal finder,
+but the results between the rest of the facets was "additive". Though this has since been changed to have an "additive" relationship between all of the facets.
+
+Normally facets are defined in the "details" hash of the content item. "details" are updated every time the content item is published via the "put_content" method in publishing-api. After being published, the content item is reindexed in search with this facet information.
+
+This meant that to be able to tag content to facets in the business finder, an interface would need to be added to every publishing app.
+
+The quick solution to this problem was to bypass the publishing pipeline completely, and instead update the facet information in search directly.
+
+Simply put, content designers would update a CSV file with a list of content and the business finder facets it needed to be tagged to. Developers would upload this CSV file and then run a script to tag the content in search.
+The script would check whether it already knew about the tags, and if it didn't, would add them. It also triggered an email every time a tag was added.
```

Comment:
I think this should be _It triggered an email every time a new item of content was added to the business finder._